corehq/apps/app_manager/models.py

```diff
@@ -5755,6 +5769,20 @@ def for_app(cls, app):
 
     def save(self, *args, **kwargs):
         LatestAppInfo(self.app_id, self.domain).clear_caches()
+
+        # Save to SQL
+        model, created = SQLGlobalAppConfig.objects.update_or_create(
+            domain=self.domain,
+            app_id=self.app_id,
+            defaults={
+                'apk_prompt': self.apk_prompt,
+                'app_prompt': self.app_prompt,
+                'apk_version': self.apk_version or LATEST_APK_VALUE,
+                'app_version': self.app_version or LATEST_APP_VALUE,
+            }
+        )
+
+        # Save to couch
         super(GlobalAppConfig, self).save(*args, **kwargs)
```

Comment:
I guess you didn't write this line, but I noticed it could be shortened now to what in python3 is the more idiomatic (and I think in weird edge cases better-behaving?) `super().save(*args, **kwargs)`.